tests/tests_unit/test_api/test_labels.py

```diff
@@ -80,3 +80,9 @@ def test_load_list(self):
         assert Label._load_list(None) == None
         labels = [{"externalId": "a"}, "b", Label("c"), LabelDefinition("d")]
         assert Label._load_list(labels) == [Label("a"), Label("b"), Label("c"), Label("d")]
+
+    def test_list_with_dataset_ids(self, mock_labels_response):
+        LABELS_API.list(data_set_ids=[1], data_set_external_ids=["x"])
+        assert [{"id": 1}, {"externalId": "x"}] == jsgz_load(mock_labels_response.calls[0].request.body)["filter"][
+            "dataSetIds"
+        ]
```

Comment:
Also add dataSetId to the mocked response and assert that the result from .list() contains it.